src/components/toolbar-button.tsx

```diff
@@ -38,6 +38,7 @@ export const ToolbarButtonComponent: React.FC<IToolbarButtonProps> =
   };
 
   const handleClick = (e: React.MouseEvent<HTMLDivElement>) => {
+    if (isTileTool && isDisabled) return;
```

Comment:
Judging by the "renders disabled select tool" test in src/components/toolbar-button.test.tsx, we want at least some disabled buttons to have their click handler triggered.

Is that right? If so, is it OK to disable click handlers for all tile tool buttons like this?

Reply:
It does seem a bit odd to allow a disabled button to be clicked, but maybe I'm just missing context. What you have seems ok to me. My only possible suggestion is to make the check tighter, so instead of `isTileTool` something like `id !== "select"`.

Reply:
I'm not sure why we're allowing the click handler to be called for a disabled select tool, either. The only situation in which I recall that making sense in other contexts is when it was considered useful to put up an alert explaining to the user why the button is disabled, but that doesn't apply here.

Reply:
I've been vacillating on this. I'm tempted to just leave it as `if (isTileTool && isDisabled) return`.

@tealefristoe, I see the point of replacing `isTileTool` with `id !== "select"` and think that makes sense. Thanks for the suggestion. But then I imagine myself having to explain that change to somebody in the future, and my only answer will be that it was to satisfy a test I didn't understand :) 

It seems like there's also a good chance I could change it to `if (isDisabled) return`, then change the test to expect the click handler _not_ to have been called, and everything would be fine. Judging by what @kswenson said and what I can see in the code, it seems like that should be safe. And hopefully QA would detect any unforeseen problems it might cause...but I'm not sure and should err on the side of caution.

I think I'll just leave it as is and merge this in tomorrow unless somebody chimes in with a stronger opinion before then.